src/pages/K8s.vue

```diff
@@ -169,7 +169,9 @@ export default {
       return os.cpus().length;
     },
     cannotReset() {
-      return ![K8s.State.STARTED, K8s.State.ERROR].includes(this.state);
+      const { STARTED, ERROR, DISABLED } = K8s.State;
+
+      return ![STARTED, ERROR, DISABLED].includes(this.state);
```

Comment:
`reset` also has a thing dealing with the state; if the state is `DISABLED`, then we shouldn't need a wipe there either, right? (Of course, the text is incorrect in that case.)